dotcom-rendering/makefile

```
@@ -19,6 +19,7 @@ endef
 
 riffraff-bundle: clean-dist build
 	$(call log, "creating riffraff bundle")
+	@yarn synth
```

Would this be
```suggestion
	@yarn run cdk:synth
```
?

I'm a little unclear as to what this is doing?

---

This builds the cloudformation templates used for deployment, not necessary if we want to split up this PR!